tools/bazel.rc

```diff
@@ -4,6 +4,14 @@
 # Therefore, this option is disabled until gRPC works with bzlmod.
 common --enable_bzlmod=false
 
+# gRPC build will break under Bazel 8 because Bazel 8 disables workspace by default.
+# To prevent this, we've enabled workspace again until gRPC fully transitions to Bzlmod.
```

Comment:
Suggest adding a TODO here so it doesn't get lost.

Reply:
Good point but I've already started; https://github.com/grpc/grpc/pull/38456